src/Windows/Avalonia.Win32/WindowImpl.AppWndProc.cs

```diff
@@ -17,6 +18,7 @@ public partial class WindowImpl
         protected virtual unsafe IntPtr AppWndProc(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam)
         {
             const double wheelDelta = 120.0;
+            const long UiaRootObjectId = -25;
```

Comment:
Is the -25 arbitrary here?

Reply:
This is from the Win32 API: https://github.com/tpn/winsdk-10/blob/master/Include/10.0.16299.0/um/UIAutomationCoreApi.h#L52

Reply:
I see, good to know. Could add a comment for this, but I think it's fine as is.